README.md

```diff
@@ -229,7 +229,14 @@ Relay [specification for mutations](https://facebook.github.io/relay/graphql/mut
 
 ### Connection
 
-You can use `@GraphQLConnection` annotation to make a field iterable in adherence to Relay [Connection specification](https://facebook.github.io/relay/graphql/connections.htm).
+You can use `@GraphQLConnection` annotation to make a field iterable in adherence to Relay [Connection specification](https://facebook.github.io/relay/graphql/connections.htm).\
+If a field is annotated with the annotation, the associated dataFetcher must return an instance of `PaginatedData`.\
+The `PaginatedData` class holds the result of the conneciton:
+1. The data of the page
+2. Whether or not there is a next page and a previous page
+3. A method that returns for each entity the encoded cursor of the entity (it returns string)
+
+NOTE: because `PropertyDataFetcher` and `FieldDataFetcher` can't handle connection, this annotation cant be used on a field that doesn't have a dataFetcher
```

Comment:
I think this limitation is quite ok, but just for information, what is the issue with Property/FieldDataFetcher ?

Reply:
Well, they don't know how to build connection (because it's not an Scalar type)